python/modules/flow_control.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/python
+
+import os, logging
+import modules.file_managment as FM
+
+logger = logging.getLogger("flow control")
+logger.info("Executing flow_control module.")
+
+
+def compare_args(arg1, arg2, comparison):
+    if comparison == "equal" or comparison == "==":
+        if arg1 == arg2:
+            return True
+    elif comparison == "different" or comparison == "!=":
+        if arg1 != arg2:
+            return True
+    elif (comparison == "greater" or comparison == ">" or 
+            comparison == "less" or comparison == "<"):
+        if _greater_or_less(arg1, arg2, comparison):
+            return True
+    else:
+        logger.error("Unknown for TASK_IF comparison type: %s", comparison)
+    return False
+
+def _greater_or_less(arg1, arg2, comparison):
+    try:
+        arg1 = int(arg1)
+        arg2 = int(arg2)
+    except:
+        logger.error("Cannot compare if given parameter is greater/less than "
+        "another when given arguments are not integers: %s, %s", arg1, arg2)
+        return False
+    if comparison == "greater" or comparison == ">":
+        if arg1 > arg2:
+            return True
+    else:
+        if arg1 < arg2:
+            return True
+    return False
+
+def create_ele_list(input_path, wells_params, used_value):   
+    """
+    Arguments:
+    - input_path - path to directories
+    - wells_params - list of base parameters of chosen wells set
+    - used_value - value defying if given dir name reflects well tag or id
+    Returns
+    - verified_dirst- list of dirs which parameters are present 
```

Comment:
I changed comments. 
Used_value wasn't remove according to our discussions and previous comments.